marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/entities/search/EntitySearchManagerTest.java

```diff
@@ -316,4 +312,22 @@ private void validateSearchWithTransform(String databaseType) {
                 "entityProperties so that the UI knows what structured values to show for each entity instance");
         assertTrue(node.get("results").get(1).has("entityProperties"));
     }
+
+    private void validateAllDataSearch(String databaseType) {
+        DatabaseClient client = databaseType.equalsIgnoreCase("staging") ? getHubClient().getStagingClient() : getHubClient().getFinalClient();
```

Comment:
Given that this client is used after runAsHubCentralUser is called, I think you should move this line to after runAsHubCentralUser. Looking at the code, it's not clear what user is associated with this DatabaseClient, but it looks like it's supposed to be running as an HC user.